e2e/cephfs.go

```diff
@@ -296,7 +296,7 @@ var _ = Describe("cephfs", func() {
 				}
 			})
 
-			By("checking nodeplugin deamonset pods are running", func() {
+			By("checking nodeplugin daemonset pods are running", func() {
```

Comment:
this has nothing to do with the PR, nor with the commit?

Reply:
@nixpanic mentioned or done in this commit as part of cleanup mentioned in the same ?

Reply:
if you are mentioning it becuase of "cephfs", sure, let me split these commits.

Reply:
regardless, rearranged the commits a bit.. ptal.. thanks @nixpanic